deepcell/applications/application_test.py

```diff
@@ -36,6 +36,8 @@
 
 from deepcell.applications import Application
 
+from deepcell_toolbox.deep_watershed import deep_watershed
+
```

Comment:
```suggestion
```

Reply:
For sure, I think for testing that the pipeline itself hands off correctly the one that's there covers the bases.